Content.Server/GameTicking/GameTicker.RoundFlow.cs

```diff
@@ -84,6 +85,9 @@ private void PreRoundSetup()
 
             var timeSpan = _gameTiming.RealTime - startTime;
             Logger.InfoS("ticker", $"Loaded map in {timeSpan.TotalMilliseconds:N2}ms.");
+
+            // Start playtime tracking
+            _playtimeManager.Ticking = true;
```

Comment:
It seems kind of weird to me that the playtime tracking is paused *only* when the *entire server* is in the lobby.